pandora-client-web/src/components/chatroomAdmin/chatroomAdmin.scss

```diff
@@ -63,11 +63,36 @@
 	gap: 0.5em;
```

Comment:
I think the `.backgroundSelect` needs its `max-width` and `max-height` changed to be constant, not max (`width`&`height`), as it feels really clunky when everything moves under your hand when modifying filters.
We think the following values look good on both normal screens and small ones (phones):
```css
width: min(50em, 90vw);
height: min(40em, 90vh);
```

Reply:
I didn't quite like the result for the height so I moved the dialog to the top so the selection doesn't jump around